cardano-cli/src/Cardano/CLI/EraBased/Options/Transaction.hs

```diff
@@ -78,12 +78,17 @@ pTransactionCmds era envCli =
         $ subParser "calculate-min-fee"
         $ Opt.info (pTransactionCalculateMinFee envCli)
         $ Opt.progDesc "Calculate the minimum fee for a transaction."
-    , Just
-        $ subParser "calculate-min-required-utxo"
-        $ Opt.info (pTransactionCalculateMinReqUTxO era)
-        $ Opt.progDesc "Calculate the minimum required UTxO for a transaction output."
-    , Just
-        $ pCalculateMinRequiredUtxoBackwardCompatible era
+    , caseByronOrShelleyBasedEra
```

Comment:
The goal is to not need `caseByronOrShelleyBasedEra` in the future. Required byron based functionality will be hard coded and available under the `byron` command group.